source/world/entity/Player.hpp

```diff
@@ -8,7 +8,7 @@
 
 #pragma once
 
-#include "RakNetTypes.h"
+#include "thirdparty/raknet/RakNetTypes.h"
```

Comment:
This has no business being changed- just add raknet's include path to the includes in the macOS project.